Gemfile

```diff
@@ -9,6 +9,7 @@ group :development do
   gem 'fakefs'
   gem 'simplecov'
   gem "chef-zero", "~> 15" # eval when we drop ruby 2.6
-  gem "chef", "~> 16" # eval when we drop ruby 2.6
-  gem "ohai", "~> 16" # eval when we drop ruby 2.6
-end
\ No newline at end of file
+  gem "chef", "~> 17"
+  gem "ohai", "~> 17"
+  gem "knife", "~> 17"
```

Comment:
might as well bump all of these to chef-18, this would be a major gem bump for knife-ec-backup.